.circleci/config.yml

```diff
@@ -131,6 +131,17 @@ jobs:
           name: "Build"
           command: cond_spot_run_build barretenberg-x86_64-linux-clang-assert 128
 
+  barretenberg-x86_64-linux-clang-sol:
+    docker:
+      - image: aztecprotocol/alpine-build-image
+    resource_class: small
+    steps:
+      - *checkout
+      - *setup_env
+      - run:
+          name: "Build"
+          command: cond_spot_run_build barretenberg-x86_64-linux-clang-sol 128
+
```

Comment:
Do we need 128 cores?

Reply:
i just copied the other clang builds, it certainly helps with the compile time, but is useless once the foundry stuff starts